### PR TITLE
ER-799 - GTM and Removing old GA snippet

### PR DESCRIPTION
**Pull request by sachpatel**

### CODE REVIEW

File: src/Employer/Employer.Web/Views/PartialNames.cs [hunk 1]
```diff
@@ -4,6 +4,8 @@ public static class PartialNames
     {
         public const string DisabilityConfident = "_DisabilityConfidentPartial";
         public const string GoogleAnalyticsSnippet = "_GoogleAnalyticsPartial";
+        public const string GoogleTagManagerHeadPartial = "_GoogleTagManagerHeadPartial";
```
Comment (chrisjensenuk) [inline src/Employer/Employer.Web/Views/PartialNames.cs]: can delete this `public const string GoogleAnalyticsSnippet = "_GoogleAnalyticsPartial";`

File: src/Employer/Employer.Web/appsettings.json [hunk 2]
```diff
@@ -21,7 +21,7 @@
     "ClientSecret": ""
   },
   "GoogleAnalytics": {
-    "TrackingId": "UA-83918739-16"
+    "TrackingManagerCode": "GTM-W7DMV8F"
```
Comment (chrisjensenuk) [inline src/Employer/Employer.Web/appsettings.json]: Should this be in our repo?

Comment (leewadhams) [reply src/Employer/Employer.Web/appsettings.json]: I don't suppose it matters too much as it's on the actual page isn't it?

Comment (sachpatel) [reply src/Employer/Employer.Web/appsettings.json]: Yeah it's visible.

File: src/Employer/Employer.Web/Views/PartialNames.cs (hunk 1 above)
Comment (sachpatel) [reply src/Employer/Employer.Web/Views/PartialNames.cs]: Will remove.

File: src/Employer/Employer.Web/appsettings.json (hunk 2 above)
Comment (chrisjensenuk) [reply src/Employer/Employer.Web/appsettings.json]: Of course!

**Pull request merged**